.changeset/mean-zoos-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/presentation-hierarchies": patch
+---
+
+Add `onlyIfNotHandled` flag to class based hierarchy definitions which allows to skip one if previous definitions are applied to a node.
```

Comment:
This should be a `minor`, since it's a public API addition. Also, I'd like the comment to mention which public API was changed (`createClassBasedHierarchyDefinition`), how and why, ideally with an example.

Reply:
Improved docs. Also when writing them, I've noticed that this flag is useless with custom definitions, since it never makes sense to have multiple definitions for the same node key.